### DISCUSSION TITLE
After recreating the topic through the UI it doesn't have the same number of partitions

### DISCUSSION THREAD

**Discussion by pelegz** [provectus/kafka-ui#3929] [Q&A] [answered]
Hey, we're using version 0.4.0. 

When recreating a Topic through the UI, we see that the topic isn't created with the previous number of partitions. Is this on purpose? 

Thanks.

**Comment by Haarolean** [comment, marked as answer]
Hi, 0.4.x is not supported
https://github.com/provectus/kafka-ui/blob/master/SECURITY.md
